### ISSUE TITLE
rednotebook: en_GB translation uses German word "Zurück" for "Back" and others

### ISSUE THREAD

**Issue by kathenas** [jendrikseipp/rednotebook#659]
**Describe the bug**

rednotebook: en_GB translation uses German word "Zurück" for "Back" and others.

**Expected behavior**

Show correct language words.

**Screenshots**

![Screenshot from 2022-12-31 05-54-53](https://user-images.githubusercontent.com/68813857/210126723-36cd7d1b-4155-4b79-89df-e0c173c51522.png)

**Versions:**
- RedNotebook version : 2.28.1
- Operating system and version: Debian unstable/sid

**Log output when reproducing bug**

```
philwyett@ks-gerrera:~$ rednotebook 
INFO     Using GtkSourceView 4
Adding /usr/share/rednotebook to sys.path
2022-12-31 05:49:09,269 INFO     Writing log to file "/home/philwyett/.rednotebook/rednotebook.log"
2022-12-31 05:49:09,269 INFO     System encoding: utf-8
2022-12-31 05:49:09,269 INFO     Language code: en_GB
2022-12-31 05:49:09,302 INFO     Spell checking languages: ['en', 'en_AU', 'en_CA', 'en_GB', 'en_US']
2022-12-31 05:49:09,303 INFO     Spell checking dictionaries: [('en', <Enchant: Aspell Provider>), ('en_AU', <Enchant: Aspell Provider>), ('en_CA', <Enchant: Aspell Provider>), ('en_GB', <Enchant: Aspell Provider>), ('en_US', <Enchant: Aspell Provider>)]
2022-12-31 05:49:09,512 INFO     Using LibYAML
2022-12-31 05:49:09,514 INFO     Connected Signals: [<Signals.SIGHUP: 1>, <Signals.SIGINT: 2>, <Signals.SIGQUIT: 3>, <Signals.SIGABRT: 6>, <Signals.SIGTERM: 15>, <Signals.SIGTSTP: 20>]
2022-12-31 05:49:09,518 INFO     Running in portable mode: False
2022-12-31 05:49:09,518 INFO     First Start: True
2022-12-31 05:49:09,518 INFO     RedNotebook version: 2.28.1
2022-12-31 05:49:09,519 INFO     System info: machine: x86_64, platform: Linux-6.0.0-6-amd64-x86_64-with-glibc2.36, processor: , python_version: 3.10.9, release: 6.0.0-6-amd64, system: Linux, GTK: (3, 24, 35), Glib: (2, 74, 0), PyGObject: (3, 42, 2), YAML: 6.0
2022-12-31 05:49:09,759 DEBUG    Default font: Cantarell 11
2022-12-31 05:49:09,759 DEBUG    Default size: 11.0
2022-12-31 05:49:09,938 INFO     Cloud ignore list: ['filter', 'these', 'comma', 'separated', 'words', 'and', '#tags']
2022-12-31 05:49:09,939 INFO     Cloud include list: ['mtv', 'spam', 'work', 'job', 'play']
2022-12-31 05:49:09,939 DEBUG    Start compiling regexes
2022-12-31 05:49:09,939 DEBUG    Finished
2022-12-31 05:49:09,951 DEBUG    Tray icon visible: False

(rednotebook:2534): Gtk-CRITICAL **: 05:49:09.951: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
2022-12-31 05:49:09,954 INFO     Opening journal at '/home/philwyett/.rednotebook/data'
2022-12-31 05:49:09,954 DEBUG    Starting to load files in dir "/home/philwyett/.rednotebook/data"
2022-12-31 05:49:09,955 DEBUG    Finished loading files in dir "/home/philwyett/.rednotebook/data"
2022-12-31 05:49:10,054 INFO     Adding example content on 2022-12-31
2022-12-31 05:49:10,113 DEBUG    Update the cloud
2022-12-31 05:49:10,114 DEBUG    Background color: rgb(255,255,255)
2022-12-31 05:49:10,114 DEBUG    Foreground color: rgb(0,0,0)
2022-12-31 05:49:10,121 DEBUG    Cloud updated
2022-12-31 05:49:10,132 INFO     Last backup was made 0 days ago
2022-12-31 05:49:24,565 DEBUG    Running on Linux. Is installed. Adding autostart option
2022-12-31 05:50:30,747 DEBUG    Add mathjax code: True
```

**Comment by kathenas** [comment]
Original bug report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1027411

**Comment by kathenas** [comment]
Affects fedora and EL builds also.

**Comment by kathenas** [comment]
Only appears on greater than 2.27.2.

**Comment by jendrikseipp** [comment]
Thanks for the report!

**Comment by kathenas** [comment]
Thanks for the fix. Seems to work here.

2.29 has been submitted to Debian, Fedora and EPEL.

**Comment by ashjoubert** [comment]
Thanks @jendrikseipp and @kathenas, confirmed fixed in `2.29+ds-1` on Debian/sid.

One oddity is that `/usr/share/locale/en_GB/LC_MESSAGES/rednotebook.mo` is now only 28 bytes, but I guess this makes sense because `en_GB` is the original language and there is nothing to translate. The size reduction is an improvement.

Happy New Year!
